apps/frontend/src/views/Results.vue

```diff
@@ -360,38 +360,38 @@ export default class Results extends mixins(RouteMixin, ServerMixin) {
   /**
    * The filter for charts. Contains all of our filter stuff
    */
-  get all_filter(): Filter {
+  get all_filter(): ControlsFilter {
```

Comment:
please change all snake case to camelcase in this file or wherever else you see it

Reply:
Corrected the old code